### PR TITLE
Upgrade to node 16

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -345,24 +345,24 @@ contribs/dist: .build/build-dll.timestamp
 	touch $@
 
 .build/node_modules.timestamp: package.json
-	npm install
-	mkdir -p $(dir $@)
+	npm install --no-optional
+	rm -rf ./node_modules/@types/openlayers
```
Comment (ger-benjamin) [inline Makefile:349]: Why this ?

**Review by ger-benjamin — APPROVED** [review]
Looks good to me, maybe ask also @fredj ?

File: Makefile (hunk 1 above)
Comment (sbrunner) [reply Makefile:350]: Installed from peer dependency from ol-layerswitcher and that breaks our types

Added as comment :-)

Comment (ger-benjamin) [reply Makefile:350]: :+1: thanks

**Pull request merged**